showcase/src/styles/styles.pcss

```diff
@@ -3,7 +3,5 @@ IMPORTANT: Stark styles are provided as PostCSS styles so they should be importe
 */
 
 /* Stark variables */
-@import "../../node_modules/@nationalbankbelgium/stark-ui/assets/styles/_variables.pcss";
 
 /* Stark components */
-@import "../../node_modules/@nationalbankbelgium/stark-ui/src/modules/app-logo/components/app-logo.component.pcss";
```

Comment:
I think we should keep this otherwise we'll lose the styles for app-logo component.

Reply:
Apparently, app-logo component is not used anymore into the Starter. So we can remove this for now.
Actually, app-logo component was already done before this change but now we have to refactor it for using scss instead of pcss.